# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1472156256 14400
#      Thu Aug 25 16:17:36 2016 -0400
# Node ID 88409829869c03e78517105cecf21d9e78c356a4
# Parent  4e97eaf021e4d29dc7faf77d266bedb25bdec8b7
fix doc build

* octave.texi: Don't include tips.tex or contrib.texi.
Delete detailed menu entries for those nodes.

diff --git a/doc/interpreter/octave.texi b/doc/interpreter/octave.texi
--- a/doc/interpreter/octave.texi
+++ b/doc/interpreter/octave.texi
@@ -183,18 +183,16 @@ It corresponds to GNU Octave version @va
 * Image Processing::
 * Audio Processing::
 * Object Oriented Programming::
 * GUI Development::
 * System Utilities::
 * Packages::
 * External Code Interface::
 * Test and Demo Functions::
-* Tips and Standards::
-* Contributing Guidelines::
 * Obsolete Functions::
 * Trouble::                     If you have trouble installing Octave.
 * Installation::                How to configure, compile and install Octave.
 * Emacs Octave Support::
 * Grammar and Parser::
 * Copying::                     The GNU General Public License.
 * Concept Index::               An item for each concept.
 * Function Index::              An item for each documented function.
@@ -885,33 +883,16 @@ Java Interface
 * Creating an Instance of a Java Class::
 * Handling Java Memory Limitations::
 
 Test and Demo Functions
 
 * Test Functions::
 * Demonstration Functions::
 
-Tips and Standards
-
-* Style Tips::                  Writing clean and robust programs.
-* Comment Tips::                Conventions for writing comments.
-* Function Headers::            Standard headers for functions.
-* Documentation Tips::          Writing readable documentation strings.
-
-Contributing Guidelines
-
-* How to Contribute::
-* Building the Development Sources::
-* Basics of Generating a Changeset::
-* General Guidelines::
-* Octave Sources (m-files)::
-* C++ Sources::
-* Other Sources::
-
 Trouble
 
 * Actual Bugs::                 Bugs we will fix later.
 * Reporting Bugs::
 * How To Get Help with Octave::
 * How to Distinguish Between Octave and Matlab::
 
 Reporting Bugs
@@ -989,18 +970,16 @@ Grammar and Parser
 @include system.texi
 @include package.texi
 
 @c ------------------------------------------------------------------------
 @c Appendices start here.
 
 @include external.texi
 @include testfun.texi
-@include tips.texi
-@include contrib.texi
 @include obsolete.texi
 @include bugs.texi
 @include install.texi
 @include emacs.texi
 @include grammar.texi
 @include gpl.texi
 
 @c ------------------------------------------------------------------------
